1-exercises/H-array-methods-2/exercise.js

```diff
@@ -15,8 +15,8 @@ let everyone = [
   "Swathi",
 ];
 
-let firstFive; // complete this statement
-let lastFive; // complete this statement
+let firstFive = everyone.slice(-5);// complete this statement
```

Comment:
everyone.slice(-5) works to give you `lastFive`. To get the first five, you would be starting at 0, finishing at 5, so everyone.slice(0,5)